tests/PySys/call_tedge_connect_c8y_--test_positive/run.py

```diff
@@ -0,0 +1,43 @@
+import sys
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+
+"""
+Test `tedge connect c8y --test` (successful case):
```

Comment:
Maybe here also change that line to
"Run connection test while being connected (positive case):"
It's maybe a bit more understandable for people that don't know the precise command.